src/js/components/TaskDetailComponent.jsx

```diff
@@ -86,12 +86,15 @@ var TaskDetailComponent = React.createClass({
   getIpAddresses: function () {
     var props = this.props;
     var task = props.task;
+    var ipAddresses = task.ipAddresses;
 
-    var ipAddresses = (task.ipAddresses != null && task.ipAddresses.length > 0)
-      ? task.ipAddresses.map(address => address.ipAddress)
-      : [task.host];
+    if (ipAddresses == null) {
```

Comment:
should we maybe also return `null` if ipAddresses is for some reason an empty array?